build.sbt

```diff
@@ -81,6 +82,7 @@ lazy val allOld = (project in file("allOld"))
     scalatest,
     scalatestSelenium,
     moduleMysql,
+    moduleNeo4j,
```

Comment:
You don't need to modify `allOld` project. It's only for backward compatibility.

Reply:
Done